pipelines/tasks/picard.wdl

```diff
@@ -92,6 +92,7 @@ task CollectDuplicationMetrics {
   runtime {
     docker: "quay.io/humancellatlas/secondary-analysis-picard:2.10.10-7ab16db"
     memory: "7.5 GB"
+    cpu: "2"
     disks: "local-disk 50 HDD"
```

Comment:
Hardcoded disk sizes scare me when it comes to production wdls.  We should think about dynamically sizing our wdls (not sure if now is the right/best time but in general).  There are a number of examples in other wdls, https://github.com/broadinstitute/dsde-pipelines/blob/develop/genomes_in_the_cloud/single_sample/PairedSingleSampleWf.wdl#L145 has a couple of them.  This helps with both cost and robustness of the workflow